mpi-proxy-split/lower-half/mmap64.c

```diff
@@ -119,7 +114,7 @@ MmapInfo_t*
 getMmappedList(int **num)
 {
   if (!num) return NULL;
-  *num = &numRegions;
```

Comment:
Am I missing something, or does this undo the pointer-tracking work?

Reply:
I agree! This is not correct. `*num` should hold the address of the `numMmapRegions` variable.

Reply:
Thanks!  You're right.  I forgot the '&'.  I'll fix that.